Source/items.cpp

```diff
@@ -529,17 +529,17 @@ void CalcPlrItemMin(Player &player)
 	}
 }
 
-void WitchBookLevel(int ii)
+void WitchBookLevel(Item &item)
 {
-	if (witchitem[ii]._iMiscId != IMISC_BOOK)
+	if (item._iMiscId != IMISC_BOOK)

```

Comment:
Given we bail out of the function if the item passed in is not a book we can use a more descriptive variable name. I went with `bookItem` in #2924 but `book` would be fine, no need to be repetitive :D

Reply:
Merged